tests/api/v1/aspects/get.js

```diff
@@ -133,7 +145,7 @@ describe(`api: GET ${path}`, () => {
       .set('Authorization', token)
       .expect(constants.httpStatus.OK)
       .expect((res) => {
-        expect(res.body.length).to.be.equal(TWO);
+        expect(res.body.length).to.be.equal(THREE);
```

Comment:
Need a "hierarchy" test which shows that the new field is included in the list of aspect fields returned for every sample.